modules/wizard/view-wizard-step-licenses.php

```diff
@@ -8,7 +8,7 @@
  */
 
 if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Don't access directly.
+	exit;
 }
```

Comment:
Since you did it everywhere else:
```php
defined( 'ABSPATH' ) || exit;
```

Reply:
Done in e6e41a36cf9ff7fd0a30ec02e2b1c40b0e44eeb7 and on other missing views